### ISSUE TITLE
[BUG] 3.2.7 - Animation Manager Locks up, Persist not looping still

### ISSUE THREAD

**Issue by robgrahamau** [fantasycalendar/FoundryVTT-Sequencer#249]
Even in 3.2.7 the Sequences for persist are still not looping, when you go into the Sequence manager and check the Sequences menus you get the same lock up on the previously mentioned as fixed only now with the actual effects list there.
The errors that are being punched out are as follows:

Uncaught (in promise) TypeError: store.subscribe is not a function
    at subscribe (utils.js:139:22)
    at get_store_value (utils.js:153:2)
    at sequence-manager.js:83:9
    at Object.update2 [as update] (index.js:69:7)
    at SequenceManager.RunningSequences.clearFinishedSequences (sequence-manager.js:80:36)
    at Sequences.svelte:13:38
    at run (utils.js:41:9)
    at Array.map (<anonymous>)
    at Component.js:47:48
    at flush (scheduler.js:99:5)Understand this error
2utils.js:139 Uncaught TypeError: store.subscribe is not a function
    at subscribe (utils.js:139:22)
    at get_store_value (utils.js:153:2)
    at sequence-manager.js:83:9
    at Object.update2 [as update] (index.js:69:7)
    at SequenceManager.RunningSequences.clearFinishedSequences (sequence-manager.js:80:36)
    at HTMLButtonElement.click_handler (Sequences.svelte:34:43)

rolling back to 3.2.5 fixes the persistence issue, but of course the sequences menu is borked there.

**To Reproduce**
run a persistent animation, watch it not loop, go to sequence manager, click sequence watch it error.

**Expected behavior**
that things loop and work,.

**Screenshots**

**Setup:**
 - Foundry Version: 328 Version 12 Stable
 - System Version: [DND5e 3.2.1
 - Sequencer Version: 3.2.7
 - Browser & version Chrome.

**Active modules:**
 - Sequencer
 - socketlib
 - Automated animations
 - j2Ba

alot.. but it works fine if i role it back

**Additional context**

**Comment by Haxxer** [comment]
What is the sequence you're using to make something persist? I cannot replicate it with the original ticket's sequence

**Comment by robgrahamau** [comment]
The main one is with Automated Animations, but i can replicate it with 
```
new Sequence()
  .effect()
    .file("jb2a.spirit_guardians.greenorange.ring")
    .attachTo(token)
    .persist()
  .play()
```
![image](https://github.com/fantasycalendar/FoundryVTT-Sequencer/assets/28775608/ec2b6055-7186-49da-a82b-17caeaa5cb08)

Those animations aren't looping i can't click on anything on sequence manager to get it to move or change etc. and def on 3.2.7
![image](https://github.com/fantasycalendar/FoundryVTT-Sequencer/assets/28775608/29d962dd-a4c1-4536-8b4a-ff5f7d6364da)

**Comment by Haxxer** [comment]
I see what's happened, dang merge issues, thanks

**Comment by robgrahamau** [comment]
Gotta hate it when those happen, just glad you know the cause. Sorry I should have put the macro in as well, just waking up.

**Comment by Haxxer** [comment]
Fixed in 3.2.8
